include/eve/module/real/core/function/regular/simd/x86/compress_store.hpp

```diff
@@ -137,7 +137,9 @@ namespace eve::detail
       }();
 
       top_bits mmask{mask};
-      aligned_ptr<std::uint64_t, 32> pattern_ptr{patterns[mmask.as_int() & 7].data()};
+      aligned_ptr < std::uint64_t
+                  , expected_cardinal_t<std::uint64_t>
+                  > pattern_ptr{patterns[mmask.as_int() & 7].data()};
```

Comment:
No, not expected cardinal.

THis should be `eve::fixed<4>` even on avx512.

We load specifically 4 integers